bundles/org.openhab.ui/web/src/components/sse-events-mixin.js

```diff
@@ -0,0 +1,154 @@
+        switch (event.topic) {
+          case topicAudio:
+            const topicParts = event.topic.split('/')
+            switch (topicParts[2]) {
+              case 'playurl':
+                this.playAudioUrl(JSON.parse(event.payload))
+                break
+            }
+            break
+          case topicCommand:
+            const payload = JSON.parse(event.payload)
+            const [command, ...segments] = payload.value.trim().split(/(?<!\\):/)
+            const combined = segments.join(':')
+            switch (command) {
+              case 'navigate':
+                this.$f7.views.main.router.navigate(combined)
+                break
+              case 'popup':
+              case 'popover':
+              case 'sheet':
+                if (combined.indexOf('page:') !== 0 && combined.indexOf('widget:') !== 0 && combined.indexOf('oh-') !== 0) {
+                  console.error('Action target is not of the format page:uid or widget:uid')
+                  return
+                }
+                console.debug(`Opening ${combined} in ${command} modal`)
+                let modalRoute = {
+                  url: combined + '/' + command,
+                  route: {
+                  }
+                }
+                if (command === 'popup') modalRoute.route.popup = { component: OhPopup }
+                if (command === 'popover') modalRoute.route.popup = { component: OhPopover }
+                if (command === 'sheet') modalRoute.route.popup = { component: OhSheet }
+                let modalProps = {
+                  props: {
+                    uid: combined,
+                    modalParams: {}
+                  }
+                }
+                this.closePopups()
+                this.$f7.views.main.router.navigate(modalRoute, modalProps)
+                break
+              case 'close':
+                this.closePopups()
+                break
+              case 'back':
+                this.$f7.views.main.router.back()
+                break
+              case 'reload':
+                window.location.reload()
+                break
+              case 'notification':
+                const payload = {
+                  text: segments[0],
+                  closeOnClick: true,
+                  closeTimeout: 5000
+                }
+                if (segments.length > 1) {
+                  payload.title = segments[1]
+                }
+                if (segments.length > 2) {
+                  payload.subtitle = segments[2]
+                }
+                if (segments.length > 3) {
+                  payload.titleRightText = segments[3]
+                }
+                if (segments.length > 4) {
+                  payload.closeTimeout = parseInt(segments[4])
+                }
+                this.$f7.notification.create(payload).open()
+                break
+            }
+            break
+        }
+      })
+    },
+    stopEventSource () {
```

Comment:
I have just noticed that this method is neither called inside the mixin nor in app.vue.

Should we keep it for completeness or remove it? I guess for the webpack bundle that does not matter due to webpack's optimization.

Reply:
I also noticed it was not used, but though maybe it was there also for completeness.  I don't have an opinion one way or another. 🤷

Reply:
Let‘s keep it there.